platforms/platform_db/platform_defaults/defaults_hssi.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
I believe this file isn't complete?

Reply:
In what way? Its HSSI-only, included by platform_defaults.json above. The definitions there are sufficient for the current HSSI MACs, but will have to be elaborated when the streaming interfaces are defined.

Reply:
The reason I ask is I don't see the default HSSI interface (raw_pr versus streaming) in this file. But, I see a comment which says there are two interfaces. How is the default interface specified?

Reply:
Ah, good question.

There is actually no interface defined here. The PIM only knows about classes (CCI-P, local_memory, hssi) and interfaces names within those classes. All it does is match a class/interface pair specified in a platform (e.g. hssi/raw_pr in https://github.com/OPAE/opae-sdk/blob/mcadler/plat_if_hssi/platforms/platform_db/discrete_pcie3_hssi40.json) with a request in an AFU JSON. There is no notion of an actual Verilog port defintion -- just a name.

Yesterday I realized I'd backed myself into exactly the corner I was claiming to solve. It used to be that an AFU could only ask for an interface specified in https://github.com/OPAE/opae-sdk/tree/mcadler/plat_if_hssi/platforms/afu_top_ifc_db. Then I realized I'd have to enumerate all possible combinations of interfaces there. Part of this change eliminates that requirement. An AFU can now start with an interface in afu_top_ifc_db but then add extended port requests. If you look at the intel-fpga-afu pull request you'll see the AFU JSON for the MACs and how this is done.

I haven't added definitions for HSSI flavors other than the raw HSSI to an AFU MAC yet.